app/assets/stylesheets/prism-modifications.scss

```diff
@@ -4,6 +4,7 @@
 pre.line-numbers {
     margin: 0 !important;
     padding-bottom: 1em !important;
+    border-radius: 0 !important;
```

Comment:
What's the reason for this change? I couldn't see it from the video and I can't really see the change when I compare branches)

Reply:
There's a 0.3em border-radius when you inspect the CSS. Not really visible, but decided to remove it as it might create some unexpected behaviors.